src/bug-filing/common/get-issue-details-html.ts

```diff
@@ -0,0 +1,65 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { escape } from 'lodash';
+import { EnvironmentInfo } from '../../common/environment-info-provider';
+import { CreateIssueDetailsTextData } from '../../common/types/create-issue-details-text-data';
+import { title } from '../../content/strings/application';
+import { IssueUrlCreationUtils } from './issue-filing-url-string-utils';
+
+const buildBodySection = (headingHtml: string, contentHtml: string): string => {
+    return `<div><b>${headingHtml}</b></div><div>${contentHtml}</div><br>`;
+};
+
+const buildIssueDetails = (help: string, helpUrl: string, ruleId: string): string => {
+    const helpEscapedForHtml = escape(help);
+    const ruleIdEscapedForHtml = escape(ruleId);
+
+    return `${helpEscapedForHtml}<br><a href="${helpUrl}">${ruleIdEscapedForHtml}</a>`;
```

Comment:
Hmm, looks like we're not encoding `helpUrl`. (Problem also exists in the old code.) Probably should have an `encodeURI` here.

Reply:
updated